UPDATING.md

```diff
@@ -742,6 +742,7 @@ The following operators were affected:
 * BashOperator (Not backwards compatible)
 * DockerOperator (Not backwards compatible)
 * SimpleHttpOperator (Not backwards compatible)
+* ECSOperator (Not backwards compatible)
```

Comment:
Oh @baxievski I think if we put this line here it will be confusing since version 2.0.0 has already been released. And with this in mind if you would add it to the next release > 2.0.0, it is no longer a breaking change, is it?

So I would suggest we remove it completely.

Reply:
@feluelle right, forgot that this is not really a breaking change since 2.0.0 was released in the meantime.